crates/wasi-nn/src/testing.rs

```diff
@@ -81,7 +81,7 @@ static ARTIFACTS: Mutex<()> = Mutex::new(());
 /// Return `Ok` if we find the cached MobileNet test artifacts; this will
 /// download the artifacts if necessary.
 #[cfg(feature = "openvino")]
-fn check_openvino_artifacts_are_available() -> Result<()> {
+fn check_openvino_onnx_artifacts_are_available() -> Result<()> {
```

Comment:
We can refactor all this stuff later, but for now it probably makes sense to retrieve any ONNX artifacts separately, e.g., with a `check_onnx_artifacts_are_available` function.